tutorials/install.md

```diff
@@ -50,6 +51,27 @@ installing Gazebo from source.
   brew install gz-sim<#>
   ```
 
+### Windows
+
+First, follow the [gz-cmake](https://gazebosim.org/api/cmake/4/install.html) tutorial for installing Conda, Visual Studio, CMake, and other prerequisites, and also for creating a Conda environment.
```

Comment:
https://github.com/gazebosim/docs/pull/512/ added more detail instructions for installing conda. Should we link to https://gazebosim.org/docs/ionic/install_windows_src/#install-dependencies instead?

Reply:
Great idea! I've updated this PR.